iosHyperskillApp/iosHyperskillApp/Sources/Modules/StepQuizHints/StepQuizHintsViewModel.swift

```diff
@@ -0,0 +1,31 @@
+import Foundation
+import shared
+
+final class StepQuizHintsViewModel: FeatureViewModel<
+    StepQuizHintsFeatureState,
+    StepQuizHintsFeatureMessage,
+    StepQuizHintsFeatureActionViewAction
+> {
```

Comment:
Здесь вместо табуляции 2 пробела

```suggestion
final class StepQuizHintsViewModel: FeatureViewModel<
  StepQuizHintsFeatureState,
  StepQuizHintsFeatureMessage,
  StepQuizHintsFeatureActionViewAction
> {
```